src/core/core.js

```diff
@@ -10,7 +10,27 @@ angular
     'material.core.gestures',
     'material.core.theming'
   ])
-  .config(MdCoreConfigure);
+  .config(MdCoreConfigure)
+  .run(DetectNgTouch);
+
+
+/**
+ * Detect if the ng-Touch module is also being used.
+ * Warn if detected.
+ */
+function DetectNgTouch($log) {
+  try {
```

Comment:
I think a better approach would be to use `$injector.has('$swipe')`, then you wouldn't need a try-catch.